router/cmd/boot/boot.go

```diff
@@ -80,6 +86,8 @@ func main() {
 	exitChan := make(chan os.Signal, 2)
 	signal.Notify(exitChan, syscall.SIGTERM, syscall.SIGINT)
 	<-exitChan
+	tail.Stop()
```

Comment:
okay I removed it. I was just briefly reading through tail's test code and saw instances of `tail.Stop` being called so I naively did the same. I'll keep the behavior the same as it once was.